docs/concepts/vega-chain/validator-scores-and-rewards.md

```diff
@@ -47,7 +47,7 @@ See below for how the validator score is calculated.
 > 
 > `num_validators` = actual number of validators running nodes on Vega
 > 
-> `comp_level` = value of the network parameter that defines the competition level
+> `comp_level` = value of the network parameter that defines the competition level*
```

Comment:
```suggestion
> `comp_level` = value of the network parameter that defines the competition level¹
```

Star gets removed because... markdown. Here's a proposal for an alternative way...